### PR TITLE
Setup ptokens and add pages

### LINKED ISSUE #8: [UI] [Quickstart] Home
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)

As someone interested in the idea of personal tokens, I want to learn more about them so that I may interact with them on the Gticoin platform.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: While many of us are familiar with the term "personal token", having quickstart page which describes the "why" and "how" for both buying and selling personal tokens is a great place to point beginners.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Page

Create gitcoin.co/token/quickstart and populate with information from the "Quickstart Home" content on Figma.

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

Content Drafted (see below)
Design Drafted (Figma)

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

Content for Quickstart Home: https://docs.google.com/document/d/1TENg-jNbXLJplfacV_6sjNPdZtSuYkyWCVUbd3f6_eQ/edit?usp=sharing

See Quickstart > Quickstart Home https://www.figma.com/file/3kFR9QQTEQRytJg5uKs5Zg/Gitcoin-Personal-Tokens?node-id=168%3A1

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Added new pages new pages

##### Refers/Fixes
Closes #8, #16, #17

### CODE REVIEW

**Comment by proofoftom** [comment]
This is awesome @iamonuwa, thanks for the contribution! Is this still a WIP? I noticed the Quick Start button doesn't go anywhere? Should go to `ptokens.views.faq`

**Comment by iamonuwa** [comment]
Alright, I'll update that. How about the explore button. Where does it go to? 

Also, I see lots of issues here. I'm available to squash them if you don't mind

**Comment by iamonuwa** [comment]
@proofoftom updated the link

**Comment by proofoftom** [comment]
Thanks!!
> How about the explore button. Where does it go to?

**Explore** will go to `/users`, and the *Make a Token* **Get Started** button will go to `/dashboard`

> Also, I see lots of issues here. I'm available to squash them if you don't mind

I don't mind at all. Would it make sense to create a separate PR though to keep things clean?

**Comment by iamonuwa** [comment]
Hey @proofoftom , had to send in the PR this way because they were identical. But subsequent PRs will be separate

**Comment by iamonuwa** [comment]
@proofoftom there is no dashboard link, instead explore. I had to point it to that. Sent in a fix for that.

File: app/ptokens/templates/buy_a_token.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                  </p>
+                  <p class="my-3">
+                    {%trans "Let Gitcoin’s coveted reputation act as a glowing example of your standing in the ever-evolving digital world." %} <br>
+                  </p>
+
+                  <p class="my-4">
+                    {%trans "What are you waiting for?" %} <br>
+                  </p>
+
+                  <a class="btn btn-gc-blue" href="{% url "explorer" %}">{%trans "Get Started" %}</a>
```
Comment (proofoftom) [inline app/ptokens/templates/buy_a_token.html]: Looks like the dashboard is fixed in commit f62b4e21842edd6d633340437e7ac3bb5a44f133. We can cherry pick that commit and change this to `{% url "dashboard" %}`, then this'll be good to go!

**Review by proofoftom — CHANGES_REQUESTED** [review]

File: app/ptokens/templates/buy_a_token.html (hunk 1 above)
Comment (iamonuwa) [reply app/ptokens/templates/buy_a_token.html]: Hey, I can't see the dashboard on that commit. Care to clarify?

**Comment by proofoftom** [comment]
Also, @iamonuwa would love to have you join us in our internal chat for the project! https://discord.gg/yWMcPUh

File: app/ptokens/templates/buy_a_token.html (hunk 1 above)
Comment (proofoftom) [reply app/ptokens/templates/buy_a_token.html]: I had to do a hard refresh to clear the browser cache for it to take effect

Comment (iamonuwa) [reply app/ptokens/templates/buy_a_token.html]: I don't understand what you mean by that. What did you do hard reset on?

**Pull request merged**